frontend/src/app/testQueue/AoEForm/AoEForm.tsx

```diff
@@ -276,13 +291,15 @@ const AoEForm: React.FC<Props> = ({
         <RequiredMessage />
         {patientIsOver18 && (
           <FormGroup title="Results">
-            <RadioGroup
-              legend="How would you like to receive a copy of your results?"
-              name="testResultDelivery"
-              onChange={setTestResultDelivery}
-              buttons={getTestResultDeliveryPreferences(patient.telephone)}
-              selectedRadio={testResultDelivery}
-            />
+            <div className="prime-formgroup__wrapper">
+              <RadioGroup
+                legend="Would you like to receive a copy of your results via text message?"
+                name="testResultDelivery"
+                onChange={setTestResultDelivery}
+                buttons={getTestResultDeliveryPreferences(patient.phoneNumbers)}
```

Comment:
**question:** will this be limited to display mobile numbers?

Reply:
Yes! Or, more specifically, non-landline numbers, per the [spec](https://docs.google.com/document/d/1Vk42CMKEtrFKqBbdPiz6JJ_uzB_RCsabqH2uFmwfPZE/edit).

>  If we don’t know if a number is mobile or landline, we should still display the number here.

Addressed in e772437.